packages/test/test-end-to-end-tests/src/test/gc/gcDatastoreAliased.spec.ts

```diff
@@ -92,8 +87,9 @@ describeFullCompat("GC Data Store Aliased Full Compat", (getTestObjectProvider)
 		const gcStatePostAlias = getGCStateFromSummary(summaryWithStats.summary);
 		assert(gcStatePostAlias !== undefined, "Should get gc post state from summary!");
 		assert(
-			gcStatePostAlias.gcNodes[ds1.handle.absolutePath].unreferencedTimestampMs === undefined,
-			"AliasableDataStore1 should be referenced as it is aliased and thus a root datastore!",
+			gcStatePostAlias.gcNodes[dataObject2.handle.absolutePath].unreferencedTimestampMs ===
```

Comment:
Aside - why doesn't `IDataStore` implement `IFluidLoadable` (I expected it would have a `handle` which would point to its entryPoint)

Reply:
LOL that's what `entryPoint` is.  Ok.  This is fine, reads better than `dataStore2.entryPoint?.absolutePath ?? ""` which is the direction I was curious about (getting the path directly from the dataStore not the dataObject)